packages/core/getUserAvailability.ts

```diff
@@ -202,10 +202,11 @@ export async function getUserAvailability(
         const startDate = dayjs(booking.start).startOf(filter);
         // this is parsed above with parseBookingLimit so we know it's safe.
         const endDate = dayjs(startDate).endOf(filter);
-
+        const bookingEventTypeId = booking.source?.split("-")[1];
         if (dayjs(booking.start).isBetween(startDate, endDate)) total++;
-        if (total >= limit)
+        if (total >= limit && bookingEventTypeId === eventType?.id?.toString()) {
```

Comment:
Only check OUR booking that matches the current eventTypeId - we don't care about another event type in this case